server/controllers/manage/residentProfileController.ts

```diff
@@ -24,16 +32,18 @@ export default class ResidentProfileController {
     return async (req: Request, res: Response) => {
       const { crn, placementId } = req.params
       const { user } = res.locals
+      const includeCancelled = (req.query.includeCancelled as string) === 'true'
       const placement = await this.placementService.getPlacement(req.user.token, placementId)
       const tabItems = residentTabItems(placement, activeTab)
       const pageHeading = tabLabels[activeTab].label
 
       let cardList: Array<SummaryListWithCard>
       let subHeading: string
       let sideNavigation: Array<TabItem>
+      let sectionTemplate: string
 
-      if (['sentence'].includes(activeTab)) {
-        sideNavigation = sentenceSideNavigation(subTab, crn, placementId)
+      if (activeTab === 'sentence') {
+        sideNavigation = sentenceSideNavigation(subTab as SentenceSubTab, crn, placementId)
```

Comment:
I'm not sure casting to a subtype helps here. It's only a type after all, it's not like you're getting any runtime checks.
If the function's formal parameter type is union'ed out of the various sub-types, this shouldn't be needed.